### ISSUE TITLE
Eventually "self-hosted" permanent storage server 

### ISSUE THREAD

**Issue by KeeJef** [loki-project/loki-messenger#511]
There is some need for some persistent storage for Loki messenger (6 months to 1 year) there is a document discussing this further in other channels. 

This issue is to discuss what open source Storage API we should use, for this purpose

**Comment by neuroscr** [comment]
blocked waiting on encryption decisions. This blocks my work on #415 with public chats

**Comment by KeeJef** [comment]
The server part of this has been completed and is at https://github.com/neuroscr/loki-syndicate-server , making a new task for client side stuff
